### ISSUE TITLE
create multiple sub-tasks by using * or - a

### ISSUE THREAD

**Issue by anntto** [getting-things-gnome/gtg#882]
New user and I've been trying to understand how to easily create multiple tasks. I came across this on the wiki page https://wiki.gnome.org/Apps/GTG
"Quickly create multiple sub-tasks by using * or - as if they were bulleted list" 

creating sub task with '-' followed by space works but I cant figure any way that * works.  
I assume it doesn't or is broken? Some of the screenshots in the help page "Creating and Managing Subtasks" seem to show * being used although the text itself only talks about dashes.

**Comment by anntto** [comment]
Not sure if it helps to say but using latest flatpak version on Ubuntu

This is the help page about 'Creating and Managing Subtasks" that I mentioned earlier.
 https://fortintam.com/gtg/user_manual/gtg-subtasks.html

**Comment by anntto** [comment]
Took a look into the code, I can see in task_view.py that it looks for '- ' couldn't see anything in there looking for '*' . (I'm no expert so I could be wrong)
Hence I assume the wiki page just needs updating,

**Comment by diegogangl** [comment]
Yup, IIRC I left * out to use it for simple lists in the future. The wiki needs updating

**Comment by pentux-GitHub** [comment]
Housekeeping pass: the user manual now documents the dash syntax (gtg-subtasks.page: start a new line with a dash and a space), and Diego confirmed above that the asterisk was deliberately left out for future simple lists. The documentation request looks satisfied, so I'd think this can be closed.
